docs/tutorials/jokes.md

```diff
@@ -1381,7 +1381,7 @@ https://pris.ly/d/getting-started
 
 Now that we've got prisma initialized, we can start modeling our app data. Because this isn't a prisma tutorial, I'll just hand you that and you can read more about the prisma scheme from [their docs](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference):
 
-```tsx filename=prisma/schema.prisma lines=[13-19]
+```prisma filename=prisma/schema.prisma lines=[13-19]
```

Comment:
As described in https://github.com/remix-run/remix/commit/8b3fc2ecefd621e794038c4eeb00db1006063f64#commitcomment-66400057

https://github.com/remix-run/remix/runs/5136619644?check_suite_focus=true#step:6:1140

CC/ @mcansh